registry/storage/driver/testsuites/testsuites.go

```diff
@@ -269,7 +269,7 @@ func (suite *DriverSuite) TestWriteReadStreamsNonUTF8(c *check.C) {
 	suite.writeReadCompareStreams(c, filename, contents)
 }
 
-// TestWriteReadLargeStreams tests that a 5GB file may be written to the storage
```

Comment:
@mikebrow This memory isn't necessarily being allocated in the test... I think this happening the node that stores the file in the driver.

Are we sure this the root cause?